editor/src/core/model/utopia-constants.ts

```diff
@@ -3,11 +3,15 @@ export const UTOPIA_LABEL_KEY = 'data-label'
 export const UTOPIA_ORIGINAL_ID_KEY = 'data-utopia-original-uid'
 export const UTOPIA_DO_NOT_TRAVERSE_KEY = 'data-utopia-do-not-traverse'
 export const UTOPIA_SCENE_ID_KEY = 'data-utopia-scene-id'
+export const UTOPIA_UID_PARENTS_KEY = 'data-utopia-parents'
+export const UTOPIA_UID_ORIGINAL_PARENTS_KEY = 'data-utopia-original-parents'
```

Comment:
I forgot what's the difference between `data-uid` and `data-utopia-original-uid`, can you help me here?

Reply:
The first one may have an index added to it if it's generated in a loop, the second one is the uid that is in the model.